src/PrestaShopBundle/Form/Admin/Sell/Product/Category/CategoriesType.php

```diff
@@ -41,11 +41,12 @@ class CategoriesType extends TranslatorAwareType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('product_categories', CategoriesCollectionType::class)
-            ->add('create_category', UnavailableType::class, [
-                'label' => $this->trans('Create a new category', 'Admin.Catalog.Feature'),
-                'label_tag_name' => 'h2',
-                'label_help_box' => $this->trans('If you want to quickly create a new category, you can do it here. Don’t forget to then go to the Categories page to fill in the needed details (description, image, etc.). A new category will not automatically appear in your shop\'s menu, please read the Help about it.', 'Admin.Catalog.Help'),
+            ->add('product_categories', CategoryTagsCollectionType::class)
+            ->add('add_categories_btn', ButtonType::class, [
+                'label' => $this->trans('Add categories', 'Admin.Catalog.Feature'),
```

Comment:
Hi, this seems to be new wording? Could you please tell me where it appears? 
I believe the right wording is "Add new category"

Reply:
The UX has been modified, this matches the latest modifications proposed by Tristan, the categories now look like this:

<img width="793" alt="Capture d’écran 2021-07-28 à 10 16 03" src="https://user-images.githubusercontent.com/13801017/127288637-64ff8bc0-94be-4f42-9f43-57a3662b5cea.png">

The click then opens a modal to select the categories:

<img width="283" alt="Capture d’écran 2021-07-28 à 10 16 14" src="https://user-images.githubusercontent.com/13801017/127288793-3b251c74-5ca2-48b6-b166-d541f73b79ff.png">

Reply:
Maybe the wording could be `Select categories` instead of `Add categories`

Reply:
Oh yes, sorry, I see, thank you. I think "Add categories" is fine because if I am not mistaken, you also have the possibility to add new categories: 
![image](https://user-images.githubusercontent.com/70583503/127880176-061b6173-b015-4e9a-9433-d08646d40aaa.png)